app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,22 @@
+class SessionsController < ApplicationController
+  def new
+    @admin = Admin.new
+  end
+
+  def create
+    admin = Admin.new(admin_params)
+    if admin.save
+      flash[:notice] = "You are successfully logged in"
+      redirect_to admins_dashboard_index_path
```

Comment:
Same question as above... when an admin logs in, where do they go? What is the admin_dashboard_index_path?